src/audiomixerboard.h

```diff
@@ -218,7 +218,9 @@ class CAudioMixerBoard :
 
     void        SetChannelLevels ( const CVector<uint16_t>& vecChannelLevel );
 
-    void        SetRecorderState ( const ERecorderState newRecorderState );
+    void            SetRecorderState ( const ERecorderState newRecorderState );
```

Comment:
Thanks for the changes! It still looks a bit odd: The function names are not aligned. So you'd need to add spaces to every other function so that all names are over each other.